test/integration/remote_installer_int_test.go

```diff
@@ -25,6 +30,25 @@ func (suite *RemoteInstallIntegrationTestSuite) TestInstall() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
+	// Setup running the remote installer in restricted powershell mode.
+	if runtime.GOOS == "windows" {
+		getPolicy := func() string {
+			policy, err := exec.Command("powershell.exe", "Get-ExecutionPolicy").CombinedOutput()
```

Comment:
Would using `ts.SpawnCmd` make this only affect the current shell? I imagine it wouldn't because this is likely a system-wide setting...

Reply:
It's a system-wide setting. I could play around with flags passed to `Set-ExecutionPolicy` to maybe affect the current session, but that doesn't reflect the nature of the test, which is to run in a restricted environment (presumably system-wide).